README.md

```diff
@@ -32,6 +33,31 @@ To build the app from source:
 - Eclipse 3.6.2 or higher (Eclipse for Android recommended)
 - Android Support Library
 
+## Release Notes
+
+### 3.0.0
+
+#### What's New in Squad Leader 3.0.0
+* Uses ArcGIS Runtime 10.2.4.
+  * If you built Squad Leader with a previous version of ArcGIS Runtime, you need to follow the step in [Building with Eclipse](#building-with-eclipse) below about copying the MIL-STD-2525C symbol dictionary into your clone. ArcGIS Runtime 10.2.4 will not work with older versions of the symbol dictionary.
+* Requires Android 4.0.1 or higher
+* Viewshed analysis
+    * Requires Android 4.1 (Android API level 16) or higher
+    * Viewshed requires elevation data on the device. See the [User Guide](documentation/UserGuide.md) for configuration details.
```

Comment:
Very minor, you could also link directly to that section: ex: https://github.com/garys-esri/squad-leader-android/blob/master/documentation/UserGuide.md#configuring-viewshed-analysis